scan-command/src/main/java/io/ballerina/scan/utils/Constants.java

```diff
@@ -61,6 +61,8 @@ public class Constants {
     static final String RULE_KIND_COLUMN = "Rule Kind";
     static final String RULE_DESCRIPTION_COLUMN = "Rule Description";
     static final String[] RULE_PRIORITY_LIST = {"ballerina", "ballerina/", "ballerinax/", "wso2/"};
+    public static final String MAIN_FUNCTION = "main";
+    public static final String INIT_FUNCTION = "init";
```

Comment:
We can have these in the visitor/analyzer instead, since they are required just there (at least for now).

Reply:
Yes but I think it is better to save constants in a one place